src/uct/ib/base/ib_md.h

```diff
@@ -15,7 +15,7 @@
 #include <ucs/stats/stats.h>
 
 #define UCT_IB_MD_MAX_MR_SIZE        0x80000000UL
-#define UCT_IB_MD_PACKED_RKEY_SIZE   sizeof(uint64_t)
+#define UCT_IB_MD_PACKED_RKEY_SIZE   2 * sizeof(uint64_t)
```

Comment:
why needed?

Reply:
This is needed to reflect the change in `uct_ib_md_packed_mkey_t`: we added a new `md_uuid` field to it, and the structure size increased by 8B.
```C
typedef struct uct_ib_md_packed_mkey {
    uint64_t md_uuid;  << newly added field
    uint32_t lkey;
    uint16_t vhca_id;
} UCS_S_PACKED uct_ib_md_packed_mkey_t;
```
And uuid field was added as a way to distinguish stale entries in alias cache on DPU side. Here is our discussion: https://github.com/openucx/ucx/pull/9752#discussion_r1711229304

Reply:
1. does it break wire compatibility for non-xGVMI use cases?
2. can we use sizeof() instead of such define?
3. wasn't the previous size 6 bytes and not 8?

Reply:
I have pushed the backward compatible approach described in p1